Makefile

```diff
@@ -155,6 +155,24 @@ ELECTRON_BUILDER_OPTIONS += $(DISABLE_UPDATES_ELECTRON_BUILDER_OPTIONS)
 endif
 
 # ---------------------------------------------------------------------
+# Publish
+# ---------------------------------------------------------------------
+
+ifdef PUBLISH
+ifndef GH_TOKEN
+$(error No GitHub token found (GH_TOKEN is not set))
+endif
+ELECTRON_BUILDER_OPTIONS += --publish always
+ifeq ($(RELEASE_TYPE),production)
+# We want a human to push the final red button
+ELECTRON_BUILDER_OPTIONS += --draft
+endif
+ifeq ($(RELEASE_TYPE),snapshot)
```

Comment:
Given that `RELEASE_TYPE` can't be both `production` and `snapshot` at the same time, you could put this within an else clause https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html